docs/AIR.md

```diff
@@ -87,7 +87,7 @@ Examples:
 (mismatch <variable> <variable> <instruction>)
 ```
 
-- executes the instruction iff variables are equal/notequal
+- executes the instruction if variables are equal/notequal
```

Comment:
Hi! Thanks for the corrections!
I think `iff` means here "if and only if", quite popular abbreviation. It may be the case above too.

Reply:
Ohh sorry about this, I changed it back.